src/framework/state/spec.ts

```diff
@@ -0,0 +1,139 @@
+import { CardID, Color, Spec } from "../types";
```

Comment:
You had mentioned that you weren't sure if these relationships should be hardcoded. While it's fine for them to be, this file should probably be in `data` instead of `framework`

Reply:
Moved.